packages/internal-lib-build/bin/internal-lib-build.js

```diff
@@ -7,6 +7,7 @@
  */
 /* eslint-disable @typescript-eslint/no-var-requires */
 const p = require('path')
+const fs = require('fs')
```

Comment:
🔫

Reply:
Thanks... I wonder why the linter isn't catching these 🤔